src/ostorlab/runtimes/local/models/models.py

```diff
@@ -386,12 +384,13 @@ class AgentGroup(Base):
     )
 
     @staticmethod
-    def create(name: str, description: str) -> "AgentGroup":
+    def create(name: str, description: str, agents: Any) -> "AgentGroup":
         """Persist the agent group in the database.
 
         Args:
-            key: Agent group key.
+            name: Agent group key.
```

Comment:
Why this change?

Reply:
its not a change, its just a correction

Reply:
just to adapt with RE, we have the name in the model, and the key will be resolved in the graphene type